src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15030,13 +15044,14 @@ namespace UnitarySystems {
                         HeatingCoils::SimulateHeatingCoilComponents(state,
                                                                     CompName,
                                                                     FirstHVACIteration,
-                                                                    DataLoopNode::SensedLoadFlagValue,
+                                                                    PartLoadFrac,
                                                                     CompIndex,
                                                                     QCoilActual,
                                                                     SuppHeatingCoilFlag,
                                                                     FanOpMode,
                                                                     PartLoadFrac); // QCoilReq= 0.0d0,  &
-                        if (!(this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingDesuperheater)) {
+                        if (!(this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingDesuperheater) &&
+                            (this->m_DesignSuppHeatingCapacity > 0)) {
                             PartLoadFrac = QCoilActual / this->m_DesignSuppHeatingCapacity;
```

Comment:
Protecting divide by 0. I'll have to think this one through. Do we always know the design supp heat capacity for gas/elec? I think so. What if that coil sizes to 0? We could get PLR = 1 if there is a load from the else below? I guess not if coil size = 0 because QCoilActual should also be 0.

Reply:
If PLR = 0 then this IF block isn't even needed.